### ISSUE TITLE
[feature request] Block Voice/Audio streams

### ISSUE THREAD

**Issue by AlexMihin** [coturn/coturn#912]
Is there a possibility (perhaps undocumented feature) to restrict Video/Audio streams from being processed by TURN server?

**Comment by AlexMihin** [comment]
up

**Comment by ggarber** [comment]
No, that option does not exist.   The type of content sent is opaque for coturn.   You could fork it and add a simple check for the RTP header to reject packets if you want.
